scalableminds/webknossos
# include created timestamp field in user json

### Steps to test:
- with network tab open, go to administration→users
- in the response json for `api/users`, there should be a `created` timestamp field

### Issues:
- contributes to #2928 

------
